src/WebJobs.Script/Binding/Http/HttpTriggerAttributeBindingProvider.cs

```diff
@@ -346,6 +392,59 @@ public override string ToInvokeString()
                     return _invokeString;
                 }
             }
+
+            private class HttpResponseValueBinder : StreamValueBinder
+            {
+                private readonly ParameterInfo _parameter;
+                private readonly HttpResponseMessage _response;
+                private readonly string _invokeString;
+
+                public HttpResponseValueBinder(ParameterInfo parameter, HttpResponseMessage response, string invokeString)
+                    : base(parameter)
+                {
+                    _parameter = parameter;
+                    _response = response;
+                    _invokeString = invokeString;
+                }
+
+                public override async Task<object> GetValueAsync()
+                {
+                    if (_parameter.ParameterType == typeof(HttpResponseMessage))
+                    {
+                        return _response;
+                    }
+                    else if (_parameter.ParameterType == typeof(object))
+                    {
+                        // for dynamic, we read as an object, which will actually return
+                        // a JObject which is dynamic
+                        return await _response.Content.ReadAsAsync<object>();
+                    }
+
+                    return await base.GetValueAsync();
+                }
+
+                protected override Stream GetStream()
+                {
+                    Task<Stream> task = _response.Content.ReadAsStreamAsync();
+                    task.Wait();
```

Comment:
I copied it from HttpRequestValueBinder class in the same file. Can you think of a scenario where this could be problematic?